.gitignore

```diff
@@ -5,6 +5,7 @@ node_modules/
 builds/
 
 dictionaries/
+dict/
```

Comment:
Why is this added?

Reply:
didn't mean to commit that, i locally have some dict converters in that folder